console/web/src/demo/demo.css

```
@@ -0,0 +1,64 @@
+   stylesheet stays the product's. Colors come from the theme tokens; only
+   the motion lives here. */
+
+/* Shine sweeping across the CTA pill, in the pill's own text color. */
+.demo-shine::after {
+  content: "";
+  position: absolute;
+  inset: 0;
+  background: linear-gradient(
+    100deg,
+    transparent 30%,
+    currentColor 50%,
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)